### ISSUE TITLE
[Bug] Bladebit - Chia Not farming compressed plots, but no errors or warnings in client or log.

### ISSUE THREAD

**Issue by Voodoo-Chia** [Chia-Network/chia-blockchain#15314]
### What happened?

This is running ChiaSetup-1.8.1rc2-dev34.exe on win10 pro 22H2
GPU GTX 1050, nvidia driver version 516.94 (installed by Windows when GPU first detected in system)
CPU: i3-6100
Farming compressed plots made with Bladebit_cuda alpha_3.1 (140TiB C7, 140TiB C4)

(Updated driver fixes all the below, but the problem here is the lack of errors in the Chia client/log.)

With above driver installed and setting the client to use_GPU_harvesting"=true in the chia config, The GUI starts without any issue. Plots are detected, none failed to open. Plots are passing the filter at the expected rate and lookup times are reported with an average of 0.5 seconds. No error or warning in the debug log (loglevel=INFO).  

So all looks good from both the Chia GUI and the Chia debug log, except that the GPU is idle and in fact not doing any decompression. 
When joining a pool however, it is discovered that the plots are in fact not being farmed at all (see log output) becasue no partials arrive at the pool.

So an unsupported Nvidia driver causes the client to falsely report that it is farming without issue, while it is in fact not. No errors or warning. 

### Version

ChiaSetup-1.8.1rc2-dev34.exe

### What platform are you using?

Windows

### What ui mode are you using?

GUI

### Relevant log output

```shell
2023-05-16T20:43:56.478 harvester chia.harvester.harvester: ERROR    Exception fetching full proof for H:\C4\plot-k32-c04-2023-04-23-03-58-605aca67d2138e457a4e05ed7fd773d192e987163afb2f2efc9ac763b36b9132.plot. GRResult is not GRResult_OK.
2023-05-16T20:43:56.481 harvester chia.harvester.harvester: ERROR    File: H:\C4\plot-k32-c04-2023-04-23-03-58-605aca67d2138e457a4e05ed7fd773d192e987163afb2f2efc9ac763b36b9132.plot Plot ID: 605aca67d2138e457a4e05ed7fd773d192e987163afb2f2efc9ac763b36b9132, challenge: 879a467882475fc61ec014c3860dcc9172acef3217af38d38ba7fab43fdb897c, plot_info: PlotInfo(prover=<chiapos.DiskProver object at 0x0000019692F830F0>, pool_public_key=None, pool_contract_puzzle_hash=<bytes32: xxx>, plot_public_key=<G1Element xxx>, file_size=88876675072, time_modified=1682215782.64)
```

**Comment by wallentx** [comment]
Can you please try with Nvidia drivers 531.79?

**Comment by Voodoo-Chia** [comment]
Yes that driver (was*) working, but the main issue is the lack of errors.

*The Nvidia driver 531.79 actually crashed this morning (nvlddmkm), see below error message from event viewer.
 After the crash, the same situation ensued. Chia client reports everything as ok, no errors, no warnings, plots passing the filter as expected. But farming has actually stopped. 
Restarted the Chia GUI, and farming resumed again correctly. 

"The description for Event ID 0 from source nvlddmkm cannot be found. Either the component that raises this event is not installed on your local computer or the installation is corrupted. You can install or repair the component on the local computer.

If the event originated on another computer, the display information had to be saved with the event.

The following information was included with the event: 

\Device\Video24
Resetting TDR occurred on GPUID:100

The message resource is present but the message was not found in the message table"

**Comment by MumfMeisterT** [comment]
Moving this issue to Bladebit Repo
